pages/manual/manual-gettingstarted-build.md

```diff
@@ -332,11 +313,25 @@ Note that it may in some cases be necessary to run the compile or the binary fro
         cmake -S inviwo --preset ninja-user-apt -DIVW_MODULE_SOMEMODULE=ON
     + Finally compile inviwo by running `ninja` in the build folder `~/Inviwo/builds/xcode-user`
 
-### Comments
 
-{% include tip.html content="
-Unless you specifically need to debug the application, we recommend setting the build mode to `RelWithDebInfo` for good performance, while still getting reasonable stacktraces for debugging and error reporting.
+## Updating Master
+To update Inviwo to the latest version, use your graphical git client to pull the latest master of the Inviwo repository and, optionally, the Inviwo modules repository. Alternatively, run the following commands in the `base` directory:
+```bash
+cd inviwo
+git pull
+cd ../modules
+git pull
+```
+In case you are using vcpkg, make sure to update vcpkg as well and set it to the correct baseline, see [Binary caching with vcpkg](#binary-caching-with-vcpkg).
```

Comment:
Think we should have vcpkg as required. 
No one will figure out how to build without vcpkg.